core/storage/email/gae_models_test.py

```diff
@@ -41,6 +43,29 @@ def _generate_hash_for_tests(
                 _generate_hash_for_tests,
                 email_models.SentEmailModel))
 
+    def test_duplicate_mails_detected(self):
+        email_models.SentEmailModel.create(
+            'recipient_id', 'recipient@email.com', 'sender_id',
+            'sender@email.com', feconf.EMAIL_INTENT_SIGNUP,
+            'Email Subject', 'Email Body', datetime.datetime.utcnow())
+
+        email_models.SentEmailModel.create(
+            'recipient_id', 'recipient@email.com', 'sender_id',
+            'sender@email.com', feconf.EMAIL_INTENT_SIGNUP,
+            'Email Subject', 'Email Body', datetime.datetime.utcnow())
+
+        duplicated = email_models.SentEmailModel.check_duplicate_message(
+            'recipient_id', 'Email Subject', 'Email Body')
+
+        self.assertEqual(duplicated, True)
+
+        not_duplicated = email_models.SentEmailModel.check_duplicate_message(
+            'recipient_id', 'Other Email Subject', 'Other Email Body')
+
+        self.assertEqual(not_duplicated, False)
+
+
+
```

Comment:
one empty line after function in a class is sufficient, remove extra new lines.

Reply:
Done